lua/telescope-alternate/presets.lua

```diff
@@ -1,34 +1,51 @@
 local M = {}
 
 M.rails = {
-  { "app/models/(.*).rb", {
-    { "app/controllers/**/*[1:pluralize]_controller.rb", "Controller" },
-    { "app/views/[1:pluralize]/*.html.erb", "View" },
-    { "app/helpers/[1]_helper.rb", "Helper" },
-    { "app/serializers/**/*[1]_serializer.rb", "Serializer" }
+  { 'app/models/(.*).rb', {
+    { 'app/controllers/**/*[1:pluralize]_controller.rb', 'Controller' },
+    { 'app/views/[1:pluralize]/*.html.erb', 'View' },
+    { 'app/helpers/[1]_helper.rb', 'Helper' },
+    { 'app/serializers/**/*[1]_serializer.rb', 'Serializer' },
+    { 'spec/requests/**/*[1:pluralize]_spec.rb', 'Request Test' },
+    { 'spec/models/**/*[1:singularize]_spec.rb', 'Model Test' },
+    { 'spec/factories/**/*[1:pluralize].rb', 'Factory' }
   } },
-  { "app/controllers(.*)/(.*)_controller.rb", {
-    { "app/models/**/*[2:singularize].rb", "Model" },
-    { "app/views/[1][2]/*.html.erb", "View" },
-    { "app/helpers/**/*[2]_helper.rb", "Helper" },
-    { "app/serializers/**/*[2]_serializer.rb", "Serializer" }
+  { 'app/controllers(.*)/(.*)_controller.rb', {
+    { 'app/models/**/*[2:singularize].rb', 'Model' },
+    { 'app/views/[1][2]/*.html.erb', 'View' },
+    { 'app/helpers/**/*[2]_helper.rb', 'Helper' },
+    { 'app/serializers/**/*[2]_serializer.rb', 'Serializer' },
+    { 'spec/requests[1]/[2]*_spec.rb', 'Request Test' },
+    { 'spec/factories/**/*[2].rb', 'Factory' }
   } },
-  { "app/views/(.*)/(.*).html.(.*)", {
-    { "app/controllers/**/*[1]_controller.rb", "Controller" },
-    { "app/models/[1:singularize].rb", "Model" },
-    { "app/helpers/**/*[1]_helper.rb", "Helper" },
+  { 'app/views/(.*)/(.*).html.(.*)', {
+    { 'app/controllers/**/*[1]_controller.rb', 'Controller' },
+    { 'app/models/[1:singularize].rb', 'Model' },
+    { 'app/helpers/**/*[1]_helper.rb', 'Helper' },
+  } },
+}
+
+M.rspec = {
+  { 'app/models/(.*).rb', {
```

Comment:
Maybe something like that?

{ "app/(.*).rb", { { "spec/[1]_spec.rb", "Test" } },
{ "spec/(.*).rb", { { "app/[1]_spec.rb", "Original" } }

And about the request tests, maybe is more relate to the controller only.